samples/04_gis_analysts_data_scientists/image_scene_classification_using_feature_classifier.ipynb

```diff
@@ -534,7 +534,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Add r in path:</p><p>data_path = os.path.join(os.getcwd(), r"NWPU-RESISC45")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>